app/presenters/hyrax/work_usage.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 module Hyrax
+  # TODO: - Analytics do we still need this?
```

Comment:
do we? 🍡

Reply:
There is some existing code that I wasn't sure what to do with.

Reply:
maybe we could open a ticket to do whatever work this is pointing to? this kind of comment is likely to get ignored indefinitely, not least because it doesn't provide whatever context the author had at time of writing.